man/frev.Rd

```diff
@@ -0,0 +1,32 @@
+\name{frev}
+\alias{frev}
+\alias{rev}
+\title{Fast reverse}
+\description{
+  Similar to \code{\link[base]{rev}} but \emph{much faster}.
```

Comment:
quibble: is 2x _much faster_? it's not an order-of-magnitude shift... maybe we soften "often much faster" or just "faster"?

Reply:
done